nodes/default_node.go

```diff
@@ -175,15 +174,6 @@ func (d *DefaultNode) DeleteNetnsSymlink() error {
 // Checks the name pattern that is common for VM-based nodes. Container-based nodes override this function to implement
 // custom name checks.
```

Comment:
You may want to change this comment as well as I think it could be misleading.

Btw. thanks for the link to the build in #1275.